frontend/src/app/components/AppView/AppView.tsx

```diff
@@ -135,6 +135,7 @@ function AppView(props: AppViewProps): ReactElement {
       isWideMode={wideMode}
       showPadding={showPadding}
```

Comment:
I think this would also add the header padding, which we don't want in this case. If it is embedded and has a chat input. we only want the bigger bottom padding.

Reply:
I think it would then be `addPaddingForHeader={(showToolbar || showColoredLine) && !containsChatInput}` then?

Reply:
hmm, addPaddingForHeader is adding a 3rem top padding instead of 1rem. 

<img width="394" alt="image" src="https://github.com/streamlit/streamlit/assets/2852129/a7081611-3125-49ba-98d2-15267a988e95">

It would probably would mostly work fine, but it would impact apps like the one embedded on the docs page [here](https://docs.streamlit.io/knowledge-base/tutorials/build-conversational-apps) (which doesn't have a toolbar / color line)

Reply:
I'm also fine with this solution. But I think we anyways need to refactor this a bit on how the "fullscreen" chat input works together with the AppView (-> I would love to move it to some kind of bottom root container). So the `addPaddingForChatInput` is probably more of a temporary change.

Reply:
Alright. I am fine with it. Can we add a screenshot test then?

Reply:
You can check out https://github.com/streamlit/streamlit/pull/5894 which shows an example of the iframe resizer working.

Reply:
ahh, didn't know about this one. Already added a test using `http://localhost:3000/?embed=true` which seems to work fine for checking the padding.